### PR TITLE
bitnami/rabbitmq: enable tpl for auth.tls.existingSecret

### PR DESCRIPTION
**Pull request by guyguy333**
**Description of the change**

Enable tpl in `auth.tls.existingSecret` value

**Benefits**

Tpl is useful for existingSecret. It's applied for others existing secrets but not for this one.

**Possible drawbacks**

None ?

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/rabbitmq/Chart.yaml [hunk 1]
```diff
@@ -23,4 +23,4 @@ name: rabbitmq
 sources:
   - https://github.com/bitnami/bitnami-docker-rabbitmq
   - https://www.rabbitmq.com
-version: 8.20.1
+version: 8.21.0
```
Comment (juan131) [inline bitnami/rabbitmq/Chart.yaml:26]: Please bump the patch version instead:
```suggestion
version: 8.20.2
```

**Review by juan131 — COMMENTED** [review]
Thanks so much for this enhancement @guyguy333 !! Please check my comment

File: bitnami/rabbitmq/Chart.yaml (hunk 1 above)
Comment (guyguy333) [reply bitnami/rabbitmq/Chart.yaml]: Sure. Fixed and pushed, thanks.

**Review by juan131 — APPROVED** [review]
LGTM

**Pull request merged**